reana_resources_k8s/templates/deployments/job-controller-template.yaml

```diff
@@ -20,6 +20,9 @@ spec:
         volumeMounts:
         - name: svaccount
           mountPath: "/var/run/secrets/kubernetes.io/serviceaccount"
+        env:
+        - name: STORAGE_BACKEND
```

Comment:
Maybe change environment variables to use `REANA_<COMPONENT>_*` structure here and in the other PR, so that people can do `env | grep REANA` and see what they have?  Could be useful later.

This would give something like `REANA_STORAGE_BACKEND`, `REANA_STORAGE_SHAREDDIR` etc like names.